icaruscode/CRT/CRTUtils/CRTT0MatchAlg.h

```diff
@@ -81,6 +103,37 @@ namespace icarus{
 
   };
 
+  struct  matchCand_PCA {
+    sbn::crt::CRTHit thishit;
+    double t0 = DBL_MIN;
+    double dca = DBL_MIN;
```

Comment:
This is a case where I think using DBL_MIN is OK since you're definitely updating the value later.